src/RequestProcessor.cs

```diff
@@ -209,6 +211,28 @@ internal StreamingMessage ProcessInvocationRequest(StreamingMessage request)
                 functionInfo = _functionLoader.GetFunctionInfo(request.InvocationRequest.FunctionId);
                 psManager = _powershellPool.CheckoutIdleWorker(request, functionInfo);
 
+                if (_dependencyDownloadTask != null
+                    && ((_dependencyDownloadTask.Status != TaskStatus.Canceled)
+                    || _dependencyDownloadTask.Status != TaskStatus.Faulted
+                    || _dependencyDownloadTask.Status != TaskStatus.RanToCompletion))
+                {
+                    psManager.Logger.Log(LogLevel.Information, PowerShellWorkerStrings.DependencyDownloadInProgress, null, true);
+                    psManager.Logger.Log(LogLevel.Information, PowerShellWorkerStrings.DependencyDownloadInProgress, null);
```

Comment:
Why writing this again into the system log? This message is only for user to see as a UX experience improvement, isn't it?

Reply:
System log is to appear on CLI console